applications/serial_lte_modem/doc/slm_description.rst

```diff
@@ -0,0 +1,1552 @@
+
+      .. parsed-literal::
+         :class: highlight
+
+         Hello, UDP#1!Hello, UDP#2!\ **UDP1/2 received**
+         Hello, UDP#3!Hello, UDP#4!Hello, UDP#5!\ **UDP3/4/5 received**
+
+   #. Observe the output of the Python script::
+
+         $ python client_udp.py
+
+         Sending: 'Hello, UDP#1!
+         Sending: 'Hello, UDP#2!
+         UDP1/2 received
+         ('000.000.000.00', 1234)
+         Sending: 'Hello, UDP#3!
+         Sending: 'Hello, UDP#4!
+         Sending: 'Hello, UDP#5!
+         UDP3/4/5 received
+         ('000.000.000.00', 1234)
+         Closing connection
+
+   #. Stop the server.
+
+      .. parsed-literal::
+         :class: highlight
+
+         **AT#XUDPSVR=0**
+         #XUDPSVR: stopped
+         OK
+
+TLS server
+~~~~~~~~~~
+
+The TLS server role is currently not supported.
+
+.. parsed-literal::
+   :class: highlight
+
+   **AT#XSOCKET=1,1,1,16842753**
+   #XSOCKET: (D)TLS Server not supported
+   ERROR
+
+   **AT#XTCPSVR=1,3443,16842753**
+   #XTCPSVR: TLS Server not supported
+   ERROR
+
+DTLS server
+~~~~~~~~~~~
+
+The DTLS server role is currently not supported (modem limitation).
+
+.. parsed-literal::
+   :class: highlight
+
+   **AT#XSOCKET=1,2,1,16842755**
+   #XSOCKET: (D)TLS Server not supported
+   ERROR
+
+DNS lookup
+~~~~~~~~~~
+
+1. Look up the IP address for a host name.
+
+   .. parsed-literal::
+      :class: highlight
+
+      **AT#XGETADDRINFO="www.google.com"**
+      #XGETADDRINFO: 172.217.174.100
+      OK
+
+#. Observe that you cannot look up the host name for an IP address.
+
+   .. parsed-literal::
+      :class: highlight
+
+      **AT#XGETADDRINFO="172.217.174.100"**
+      ERROR
+
+Socket options
+~~~~~~~~~~~~~~
+
+After opening a client-role socket, you can configure various options.
+
+1. Check the available values for the XSOCKETOPT command.
+
+   .. parsed-literal::
+      :class: highlight
+
+      **AT#XSOCKETOPT=?**
+      #XSOCKETOPT: (0, 1), <name>, <value>
+      OK
+
+#. Open a client socket.
+
+   .. parsed-literal::
+      :class: highlight
+
+      **AT#XSOCKET=1,1,0**
+      #XSOCKET: 2, 1, 0, 6
+      OK
+
+#. Test to set and get socket options.
+   Note that not all options are supported.
+
+   .. parsed-literal::
+      :class: highlight
+
+      **AT#XSOCKETOPT=1,20,30**
+      OK
+
+      **AT#XSOCKETOPT=0,20**
+      ERROR  // to be investigated
+
+      **AT#XSOCKETOPT=0,2**
+      #XSOCKETOPT: ignored
+      OK
+
+      **AT#XSOCKETOPT=1,2,1**
+      #XSOCKETOPT: ignored
+      OK
+
+      **AT#XSOCKETOPT=0,61**
+      #XSOCKETOPT: not supported
+      OK
+
+      **AT#XSOCKETOPT=1,61,30**
+      #XSOCKETOPT: not supported
+      OK
+
+Testing ICMP AT commands
+------------------------
+
+Complete the following steps to test the functionality provided by the :ref:`SLM_AT_ICMP`:
+
+1. Ping a remote host, for example, www.google.com.
+
+   .. parsed-literal::
+      :class: highlight
+
+      **AT#XPING="www.google.com",45,5000,5,1000**
```

Comment:
This command doesn't work for me, I only get "ERROR".
Does it make sense to use www.google.com and its IP address as example?

Reply:
It seems like something wrong in the network.
Can you try below command before connecting to network?

AT%XEPCO=0

Reply:
AT+CFUN=1 and AT+CFUN? did the trick :)